redbot/core/commands/help.py

```diff
@@ -187,6 +187,11 @@ def get_default_tagline(ctx: Context):
         description = command.description or ""
         tagline = (await ctx.bot._config.help.tagline()) or self.get_default_tagline(ctx)
         signature = f"`Syntax: {ctx.clean_prefix}{command.qualified_name} {command.signature}`"
+        signature += (
+            f"\n`Alias{'es' if len(command.aliases) > 1 else ''}: {', '.join(command.aliases)}`"
```

Comment:
This construction isn't i18n safe and cannot be used as-is.

Reply:
Ignore this for now, the i18n stuff wasn't fully handled and this can be adapted for i18n when it is.